### DISCUSSION TITLE
Am I missing the way to merge 2 or more documents or does it not exist?

### DISCUSSION THREAD

**Discussion by zombiehoffa** [jonaswinkler/paperless-ng#522] [Feature Requests]
If I am missing, it would be super useful if it was possible to select and merge multiple documents into one document (pretty much exclusively pdfs in my case). My scanner does not gracefully scan multiple widths of paper so to attach receipts together right now I had to export the two parts of the receipt, use pdfunite on it on my computer then import it back in, which is a bit painful.

**Comment by shamoon** [comment]
See https://github.com/jonaswinkler/paperless-ng/issues/335 :)
